### PR TITLE
IDEX-3193: Activate go into featura for project nodes

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**

### CODE REVIEW

File: ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/actions/GoIntoAction.java [hunk 1]
```diff
@@ -59,6 +59,6 @@ public void actionPerformed(ActionEvent e) {
     }
 
     private boolean isNodeSupportGoInto(Object node) {
-        return node != null && node instanceof Node && ((Node)node).getParent() != null && ((Node)node).supportGoInto();
+        return node != null && node instanceof Node && ((Node)node).supportGoInto();
```
Comment (vzhukovskii) [inline ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/actions/GoIntoAction.java]: checking for nullable parent was added specially, cause going into project node simply has no meaning

Comment (vzhukovskii) [inline ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/actions/GoIntoAction.java]: @vparfonov should we allow to go into project node?

Comment (AndrienkoAleksandr) [inline ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/actions/GoIntoAction.java]: But  it's has meaning when you works with few projects and you want to switch and work with one of them. By the way, Eclipse support go into for projects...

Comment (evoevodin) [inline ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/actions/GoIntoAction.java]: BTW `node != null` is useless check here

Comment (vzhukovskii) [inline ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/actions/GoIntoAction.java]: yes, you're right

Comment (AndrienkoAleksandr) [inline ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/actions/GoIntoAction.java]: node != null  and node.getParent != null this is different things...

Comment (AndrienkoAleksandr) [inline ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/actions/GoIntoAction.java]: Ok i agree with you about  node != null . I'll fix it

**Comment by vparfonov** [comment]
ok

**Pull request merged**